### PR TITLE
feat: add very basic load generator for JS SDK

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

**Comment by danielamiao** [comment]
I'm going to review this tomorrow but please do not merge without my 👍 there are a few things I wanna think through first.

File: javascript/load-gen.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+const authToken = process.env.MOMENTO_AUTH_TOKEN;
+if (!authToken) {
+  throw new Error('Missing required environment variable MOMENTO_AUTH_TOKEN');
+}
+
+const defaultTtl = 60;
+
+const loggerOptions: LoggerOptions = {
+  level: LogLevel.DEBUG,
+  // level: LogLevel.INFO,
```
Comment (danielamiao) [inline javascript/load-gen.ts:24]: NIT: pull this out into a proper comment listing all the possible log levels?

File: javascript/load-gen.ts [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      logger.info('cache already exists');
+    } else {
+      throw e;
+    }
+  }
+
+  const numAsyncWorkers = 1000;
+  const numRequestsPerWorker = 50;
+  // const numAsyncWorkers = 1;
+  // const numRequestsPerWorker = 1000;
```
Comment (danielamiao) [inline javascript/load-gen.ts:49]: Suggestion: delete comments and start with modest numbers, 10 workers and 1000 requests per worker?

**Review by danielamiao — COMMENTED** [review]
A few minor comments on the code inline, 2 high-level comments:

- A simple README on how to compile/run
- As discussed offline, please add a disclaimer in both the README and in console output that this is work in progress and not meant to be a super rigorous benchmarking tool

**Comment by cprice404** [comment]
closing in favor of https://github.com/momentohq/client-sdk-examples/pull/97

**Pull request closed without merging**